fabric-networking-api-v1/src/main/java/net/fabricmc/fabric/mixin/networking/access/CustomPayloadC2SPacketAccess.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.networking.access;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+import net.minecraft.network.PacketByteBuf;
+import net.minecraft.network.packet.c2s.play.CustomPayloadC2SPacket;
+import net.minecraft.util.Identifier;
+
+@Mixin(CustomPayloadC2SPacket.class)
+public interface CustomPayloadC2SPacketAccess {
```

Comment:
Hmm, I've noticed the naming for accessors is a bit inconsistent in fabric api.

Hook, Accessor, Access?

I'd lean towards `Accessor` but this inconsistency should be brought up in another PR to resolve it.

Reply:
imo access and accessor are both the same. the interface does give access to the original object.